src/routes/router.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { createBrowserRouter } from 'react-router-dom'
+import ProtectedRoute from './protectedRoute'
+import NavBar from '@components/Navbar'
+import Main from '@pages/Main/Main'
+import Rank from '@pages/Rank/Rank'
+import MyRecord from '@pages/MyRecord/MyRecord'
+import Login from '@pages/Login/Login'
+import AddRecord from '@pages/AddRecord/AddRecord'
+import DetailRecord from '@pages/DetailRecord/DetailRecord'
+import NotFound from '@pages/NotFound/NotFound'
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: (
+      <ProtectedRoute isPublic={true}>
+        <NavBar />
+      </ProtectedRoute>
+    ),
+    children: [
+      { index: true, element: <Main /> },
+      { path: 'rank', element: <Rank /> },
+      { path: 'myrecord', element: <MyRecord /> },
+      { path: 'setting', element: <MyRecord /> },
```

Comment:
```suggestion
      { path: 'setting', element: <Setting /> },
```
element안에 Setting임포트해서 Setting이 들어가야하는거 아닌가용??

Reply:
앗.. 오타 수정할게요